concepts/char/about.md

```diff
@@ -1,3 +1,42 @@
 # About
 
-TODO
+Rust implements the [char][char type] type to represent a single character. A `char` literal is placed within single quotes, like `'a'`.
+Each `char` is four bytes in size, which is large enough to represent most [Unicode][unicode] characters as [Unicode Scalar Values][unicode scalar].
+
+However, a [`char`][character type] is not always what we think of as a letter. There are some languages, e.g. Hindi,  that use [diacritics][diacritics],
+which are special symbols which modify the character they are attached to. Although the diacritic in Rust is a separate `char`, it is the diacritic and
+the character it modifies that we commonly think of as a letter.
+
+The term for a character and its diacritic is [`grapheme cluster`][grapheme cluster]. There are external crates that can be used to process grapheme clusters,
```

Comment:
Sorry, I just committed the backticks removal independently.

Reply:
@bobahop no need to apologize! I do not care about suggestion attribution. I realized it would be clearer for me to make the suggestion than explain in plain English.